qiskit/visualization/circuit/_utils.py

```diff
@@ -285,21 +281,18 @@ def get_wire_label(drawer, register, index, layout=None, cregbundle=True):
     return wire_label
 
 
-@deprecate_arg("reverse_bits", since="0.22.0", package_name="qiskit-terra")
-def get_condition_label_val(condition, circuit, cregbundle, reverse_bits=None):
+def get_condition_label_val(condition, circuit, cregbundle):
```

Comment:
```suggestion
@deprecate_arg(
    "reverse_bits",
    since="0.22.0",
    package_name="qiskit-terra",
    removal_timeline="in the Qiskit 1.0 release",
)
def get_condition_label_val(condition, circuit, cregbundle, reverse_bits=None):
```